### PR TITLE
Course card refactor

### PR DESCRIPTION
**Pull request by Erin007**
Take attributes out of the opaque cardData, and pass props to CourseCard individually. A few other quick clean-ups.

### CODE REVIEW

File: apps/src/templates/teacherHomepage/CourseCard.jsx [hunk 1]
```diff
@@ -120,26 +120,17 @@ const CourseCard = React.createClass({
...
           >
-            {this.tooltipContent()}
+            <span style={styles.tooltip}>
+              {i18n.assignedTo()} {sections}{ellipsis}
+            </span>
           </ReactTooltip>
         </span>
       );
     }
   },
```
Comment (Erin007) [inline apps/src/templates/teacherHomepage/CourseCard.jsx:131]: Another mini render method bites the dust!

**Comment by Erin007** [comment]
cc: @joshlory @tanyaparker

File: apps/src/templates/teacherHomepage/CourseCard.story.jsx [hunk 2]
```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 import CourseCard from './CourseCard';
 
-const EXAMPLE_CARD_DATA = {
+const ExampleCard = {
```
Comment (Bjvanminnen) [inline apps/src/templates/teacherHomepage/CourseCard.story.jsx]: nit: PascalCase generally indicates a class or enum. I would use camelCase here.

File: apps/test/unit/templates/teacherHomepage/CourseCardTest.js [hunk 3]
```diff
@@ -4,28 +4,32 @@ import { shallow } from 'enzyme';
 import  CourseCard from '@cdo/apps/templates/teacherHomepage/CourseCard';
 
 describe('CourseCard', () => {
-  const noneEnrolled = {
+  const NoneEnrolled = {
```
Comment (Bjvanminnen) [inline apps/test/unit/templates/teacherHomepage/CourseCardTest.js]: again camelCase instead of PascalCase here.

**Pull request merged**